ewc-rfc001-issue-verifiable-credential.md

```diff
@@ -501,6 +541,82 @@ Query params for the authorisation request are given below:
   </tr>
 </table>
 
+### 3.5.2 Using `scope`
+
+Below is an example of such a request using `scope` parameter. Here, the wallet gets the `scope` during discovery:
+
+```http
+GET https://my-issuer.rocks/auth/authorize?
+
+GET /authorize?
+  response_type=code
+  &scope=VerifiablePortableDocumentA1
+  &resource=https%3A%2F%2Fcredential-issuer.example.com
+  &client_id=s6BhdRkqt3
+  &code_challenge=E9Melhoa2OwvFrEMTJguCHaoeK1t8URWbuGJSstw-cM
+  &code_challenge_method=S256
+  &redirect_uri=https%3A%2F%2Fclient.example.org%2Fcb
+Host: server.example.com
+```
+
+The query params for the authorisation request with `scope` are as described below:
+
+<table>
+  <tr>
+  <td><code>response_type</code>
+  </td>
+  <td>The value must be ‘code’
+  </td>
+  </tr>
+  <tr>
+  <td><code>scope</code>
+  </td>
+  <td>A JSON string that identifies the scope value supported by the Credential Issuer for a particular Credential. The value can be consistent across multiple credential configuration objects. The Authorization Server must uniquely identify the Credential Issuer based on the scope value. The Wallet can use this value in the Authorization Request. Scope values in this Credential Issuer metadata may overlap with those in the scopes_supported parameter of the Authorization Server.
+  </td>
+  </tr>
+  <tr>
+  <td><code>state</code>
```

Comment:
The `state´ parameter is not specified in draft 13 nor in HAIP, but it is REQUIRED in the EBSI specification. We have to decide how to handle this.

Reply:
OpenID4VCI is built on top of OAuth Framework. This RFC explains it more: https://datatracker.ietf.org/doc/html/rfc6749#section-4.1.1

If it is confusing, We can remove `state´ if that makes it easier to follow.

Reply:
@lalc Should we decide to remove the dynamic credential request example in authorisation response and rather point to RFC 002 for the same ?

Reply:
Will remove it.